local-container-endpoints/config/config.go

```diff
@@ -26,6 +26,10 @@ const (
 	TDRevisionVar            = "TASK_DEFINITION_REVISION"
 	ContainerInstanceTagsVar = "CONTAINER_INSTANCE_TAGS"
 	TaskTagsVar              = "TASK_TAGS_VAR"
+
+	// Custom endpoint related
+	IAMCustomEndpointVar = "IAM_CUSTOM_ENDPOINT"
+	STSCustomEndpointVar = "STS_CUSTOM_ENDPOINT"
```

Comment:
Change these to `IAM_ENDPOINT` and `STS_ENDPOINT`

Reply:
Done.